xmodule/js/src/tabs/tabs-aggregator.js

```diff
@@ -63,8 +63,8 @@
         if ($.isFunction(onSwitchFunction)) {
           onSwitchFunction();
         }
-        this.$tabs.removeClass('current');
-        $currentTarget.addClass('current');
+        this.$tabs.attr('aria-current', 'false').removeClass('current'));
```

Comment:
Oops, I think there is a dangling parenthesis
```suggestion
        this.$tabs.attr('aria-current', 'false').removeClass('current');
```